### PR TITLE
CED-1741 checkbox

### PR DESCRIPTION
**Pull request by tomleo**
### 🔗 Linked issue

https://energysage.atlassian.net/browse/CED-1741

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Checkbox component, opted for native checkbox given the simplicity of the component, and the added complexity to adapt the bootstrap styles to the primevue checkbox component (they have a "box" `<div>` for stylized checkmarks whereas bootstrap styles a sibling label)

### 🥼 Testing

- Manual

#### 🧐 Feedback Requested / Focus Areas

-

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.
- [x] I have documented testing approach

### CODE REVIEW

**Comment by hroth1994** [comment]
👀

File: es-ds-components/components/es-form-checkbox.vue [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+const model = defineModel({
+    default: false,
+});
+defineOptions({
+  inheritAttrs: false
```
Comment (hroth1994) [inline es-ds-components/components/es-form-checkbox.vue:6]: nit: 2 spaces instead of 4

File: es-ds-docs/pages/molecules/checkbox.vue [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    <div>
+        <h1>
+            Checkbox
+        </h1>
+        <p class="mb-500">
+            Uses
+            <ds-link
+                to="https://v3.primevue.org/checkbox/"
+                target="_blank">
+                primevue checkbox
```
Comment (hroth1994) [inline es-ds-docs/pages/molecules/checkbox.vue:52]: PrimeVue checkbox isn't being used, should probably change this to say something like HTML input element

**Review by hroth1994 — COMMENTED** [review]
Nice work! Left a few comments, and noticed this odd (yet, pre-existing) behavior

https://github.com/user-attachments/assets/9eeb9b3e-bcf3-4664-94c5-681d34ae9625

File: es-ds-docs/pages/molecules/checkbox.vue (hunk 2 above)
Comment (tomleo) [reply es-ds-docs/pages/molecules/checkbox.vue]: 🤦 done via [1fb89ec](https://github.com/EnergySage/es-ds/pull/1485/commits/1fb89eccc915ebedeb2f2804673958a79d97fb8c)

File: es-ds-components/components/es-form-checkbox.vue (hunk 1 above)
Comment (tomleo) [reply es-ds-components/components/es-form-checkbox.vue]: Done via [43d3fbf](https://github.com/EnergySage/es-ds/pull/1485/commits/43d3fbfded02e77eeea20aec9712e54de7ebb828)

**Comment by tomleo** [comment]
> noticed this odd (yet, pre-existing) behavior

Yeah, I spent more time than I'd like to admit trying to fix this. I'm inclined to maybe create a follow-up ticket to address this as the API at least matches what was there prior.

**Review by hroth1994 — APPROVED** [review]
LGTM! 🌟

**Pull request merged**